test/Interpreter/foundation-bridge-error.swift

```diff
@@ -0,0 +1,22 @@
+// Check that we can access localizedDescription, which crashes in the runtime
```

Comment:
Note: this test passes without this commit in the default configuration because Foundation is autolinked into the swift-frontend. However with`--param use_os_stdlib` it was failing before and now passing.  I haven't come up with a better approach.